src/app/services/bucket/dummyBucketService.py

```diff
@@ -0,0 +1,19 @@
+from app.services.bucket.baseBucketService import BaseBucketService
+
+
+class DummyBucketService(BaseBucketService):
+    def __init__(self):
+        """Dummy class used for testing purpose.
+        """
+
+    async def create_bucket(self, bucket_name):
+        return True
```

Comment:
Didn't pay attention to that one before, perhaps it would be safer to remove this method?
I mean we can do this on the CSP administration platform, and adding this could mean that our monthly bill could multiply if someone calls the wrong method :sweat_smile: